examples/arkode/C_serial/CMakeLists.txt

```diff
@@ -59,29 +59,30 @@ set(ARKODE_examples
     "ark_kepler\;--stepper SPRK --step-mode fixed --method ARKODE_SPRK_RUTH_3_3 --tf 50 --check-order --nout 1\;exclude-single"
     "ark_kepler\;--stepper SPRK --step-mode fixed --method ARKODE_SPRK_YOSHIDA_6_8 --tf 50 --check-order --nout 1\;exclude-single"
     "ark_kepler\;\;develop"
-    "ark_kpr_mri\;\;develop"
-    "ark_kpr_mri\;0 0.002\;develop"
-    "ark_kpr_mri\;1 0.002\;develop"
-    "ark_kpr_mri\;2 0.005\;develop"
-    "ark_kpr_mri\;3 0.01\;develop"
-    "ark_kpr_mri\;4 0.002\;develop"
-    "ark_kpr_mri\;5 0.002\;develop"
-    "ark_kpr_mri\;6 0.005\;develop"
-    "ark_kpr_mri\;7 0.001 -100 100 0.5 1\;exclude-single"
-    "ark_kpr_mri\;7 0.001\;exclude-single"
-    "ark_kpr_mri\;8 0.001 -100 100 0.5 1\;exclude-single"
-    "ark_kpr_mri\;8 0.001\;exclude-single"
-    "ark_kpr_mri\;9 0.001 -100 100 0.5 1\;exclude-single"
-    "ark_kpr_mri\;9 0.001\;exclude-single"
+    "ark_kpr_mri\;0 1 0.005\;exclude-single"
```

Comment:
We do not need to move it right now, but with the number of different scenarios, this seems like its really a test code for us rather than an example code.

Reply:
Indeed.  I'll note that I only added one more to the list in this PR, but since I added an extra command-line argument to the example, I had to adjust the existing lines.  

In the future, I recommend that only a couple of versions of this example be "examples" and the rest be tests.